packages/genesys-spark-tokens/data/tokens.json

```diff
@@ -8962,10 +8962,6 @@
           "value": "{size.xl}",
           "type": "sizing"
         },
-        "fixed_width": {
-          "value": "{size.3xl}",
-          "type": "sizing"
-        },
```

Comment:
Bit confused here. I would have assumed there would have to be corresponding snapshot changes to go with this change. However the tests passed so I am suspicious that there is something wrong with our tests on github.

Reply:
I forgot to run the `new-tokens-data` command which is why the snapshots never updated but I have now. But the tests should probably fail I would assume 🤔

Reply:
Had a quick look. The github action is calling `test.ci` that calls `npm run test.ci --workspaces --if-present`.
Not every package has a `test.ci` command, the tokens package being one of them. We should add a `test.ci` script to any package that does not have it. My guess is something like `"test.ci": "npm run test",` in any package that has a `test` npm script but no `test.ci` script.

Reply:
Let me know if you want to look into the test issue in this PR or create a seperate ticket for it. If you are doing it in a new ticket I will +1 this PR.

Reply:
Will take a look at it in this PR 👍

Reply:
Actually I will do it in a separate PR  so if it occurs again in the future.